src/metrics/interview_type.py

```diff
@@ -0,0 +1,36 @@
+from article_metrics.utils import lmap
+from .logic import generic_ga_filter
+
+def query_processor_frame_1(ptype, frame):
+
+    adhoc = lmap(lambda path: "ga:pagePath==%s" % path, [
+        "/from-ancient-dna-to-decay-an-interview-with-jessica-metcalf",
```

Comment:
Could perhaps store these path values outside of the code? I see the ease of containing them here but might be easier to manage when the news items scale up in number.

Reply:
these are old urls, they're not going to change and we won't be supporting new news paths at old locations, if that makes sense ?

there are cases where there is a tremendous amount of redirects, and I have a simple parser happening for those, but for the most part they are contained within the code.

Reply:
Ok, thank you for the context, that makes sense then.